released/packages/coq-pi-agm/coq-pi-agm.1.2.8/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "yves.bertot@inria.fr"
+
+homepage: "http://www-sop.inria.fr/members/Yves.Bertot/"
+bug-reports: "yves.bertot@inria.fr"
+license: "CECILL-B"
+build: [["coq_makefile" "-f" "_CoqProject" "-o" "Makefile" ]
+       [ make "-j" "%{jobs}%" ]]
+install: [ make "install" "DEST='%{lib}%/coq/user-contrib/pi_agm'" ]
+depends: [
+  "ocaml"
+  "coq" {<= "8.19"}
```

Comment:
```suggestion
  "coq" {>= "8.17" & < "8.20~"}
```
I think this should be like for version 1.2.7, right?

Reply:
Thanks for the review.  Do you recommend to have an upper bound?  This version compiles without a deprecation warning, so there are chances it will work for several version of coq (like 1.2.6, which actually lasted for 5 versions).

I understand it will be inconvenient to potential users in 2 or 3 years from now, if like this time, I forget to check the compatibility with incoming versions.

Reply:
I forgot to say.  I think version 1.2.8 is incompatible with coq.8.17: one of the deprecated theorems in 8.19 is replaced by a theorem that does not exist (yet) in coq.8.17

Reply:
It's fine by me if you skip the upper bound and just have `>= 8.19`.